x-pack/legacy/plugins/siem/server/lib/detection_engine/rules/create_rules_stream_from_ndjson.ts

```diff
@@ -53,21 +33,6 @@ export const validateRules = (): Transform => {
   });
 };
 
-// Adaptation from: saved_objects/import/create_limit_stream.ts
-export const createLimitStream = (limit: number): Transform => {
-  let counter = 0;
-  return new Transform({
-    objectMode: true,
-    async transform(obj, _, done) {
-      if (counter >= limit) {
-        return done(new Error(`Can't import more than ${limit} rules`));
-      }
-      counter++;
-      done(undefined, obj);
-    },
-  });
-};
-
 // TODO: Capture both the line number and the rule_id if you have that information for the error message
```

Comment:
I think this comment (and the function that follows it) are now moved elsewhere, and can be deleted

Reply:
@angorayc I think the `createRulesStreamFromNdJson` definition below is dead code, but we can clean it up in a subsequent PR if you'd like.

Reply:
My bad, I shouldn't have put `createRulesStreamFromNdJson` into server/utils as this is only consumed by `import_rules_route`, I'll remove the one in server/utils and keep this one here.

Regarding to the file create_rules_stream_from_ndjson.ts in `server/utils`, I renamed it to create_stream_from_ndjson.ts.